### DISCUSSION TITLE
What was the speed for the last fifth of the lap? 🏁🏎️

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#919] [Galaxy Trivia] [answered]
During a recent Formula 1 race meeting, the fastest driver managed to complete a lap with an average speed of 150 mph. They managed to complete the first two fifths of the lap at a speed of 123 mph and the second two fifths of the lap at a speed of 164 mph.

**Comment by peterbaccaro** [comment, marked as answer]
Speed is calculated using the formula **time = distance / speed**.

If number of miles in a lap is **x** miles.

The first 2/5 are covered at 123 mph. 
Therefore, the time taken was **[(2/5) * x] / 123** hours. This can be simplified as **2x / 5 * 123**.

The next 2/5 are covered at 164 mph. 
Therefore, the time taken was **[(2/5) * x] / 164** hours. This can be simplified as **2x / 5 * 164**.

The last 1/5 is covered at y mph. Therefore, the time taken was **[(1/5) * x] / y** hours. This can be simplified as **x / 5 * y**.

The entire trip is covered at an average of 150 mph. 
Therefore we can conclude that:

**2x / 5 * 123** + **2x / 5 * 164** + **x / 5 * y** 
= **x / 150**

That means that **y = 1 / (1/30 - 2/123 - 2/164)**

Therefore the speed for the last fifth of the lap was **205 mph**.
